src/llmcompressor/pipelines/sequential/helpers.py

```diff
@@ -403,6 +405,46 @@ def match_modules(model: Module, target_names: List[str]) -> Set[Module]:
     )
 
 
+def get_targets_from_modifiers(
+    modifiers: List[Modifier], model: PreTrainedModel
+) -> Tuple[List[str], List[str]]:
+    """
+    Infer sequential targets and ignore list from modifiers list
+
+    :param model: model being calibrated
+    :param modifiers: list of modifiers being applied during calibration
+    :return: list of sequential targets and list of modules to ignore for tracing
+    """
+    # avoid circular import
+    from llmcompressor.pipelines.registry import SEQUENTIAL_MODIFIERS
+
+    sequential_modifiers = [
+        modifier for modifier in modifiers if isinstance(modifier, SEQUENTIAL_MODIFIERS)
+    ]
+
+    if len(sequential_modifiers) >= 2:
```

Comment:
After this we won't be supporting the cases where we have two GPTQ modifiers (acting on different parts of the model); now we would only allow this to be conveyed via different config_groups, is my understanding correct?

Reply:
That's a good point. Yes, as currently implemented, that recipe is disallowed because those two modifiers could have conflicting `sequential_targets`.

Supporting those kinds of recipes is a larger job, since it requires either extracting the `sequential_targets` field away from modifiers (as it should be) or moving the code that resolves `sequential_targets` out of modifiers and into this function

Reply:
Sounds good, thank you!